video_xblock/static/js/videojs-transcript.js

```diff
@@ -3,24 +3,42 @@ domReady(function() {
 
     // fire up the plugin
     var transcript = this.transcript({
-      'showTrackSelector': false
+      'showTrackSelector': false,
+      'showTitle': false,
+      'clickArea': ''
     });
 
     // attach the widget to the page
     var transcriptContainer = document.getElementById('transcript');
     transcriptContainer.appendChild(transcript.el());
 
+    // Show or hide the transcripts block in depend of the transcript state
+    transcriptContainer.style.display = this.transcriptEnabled ? 'block' : 'none';
+
+    this.on('transcriptenabled', function(){
+      transcriptContainer.style.display = 'block';
+      this.transcriptEnabled = true;
+      this.trigger('transcriptstatechanged');
+    });
+    this.on('transcriptdisabled', function(){
+      transcriptContainer.style.display = 'none';
+      this.transcriptEnabled = false;
+      this.trigger('transcriptstatechanged');
+    });
+
     this.toggleButton({
       style: "fa-cc",
       enabledEvent: "captionenabled",
       disabledEvent: "captiondisabled",
       cssClasses: "vjs-custom-caption-button vjs-control",
     });
+    var cssClasses = "vjs-custom-transcript-button vjs-control";
+    this.transcriptEnabled ? cssClasses += ' vjs-control-enabled': '';
```

Comment:
Ternary operator. Your using it semantically wrong here.
Rewrite with `if (...) {...}`